lib/MusicBrainz/Server/EditQueue.pm

```diff
@@ -149,6 +158,28 @@ sub _process_open_edit
         $self->log->debug("Applying edit #$edit_id\n");
         unless ($self->dry_run) {
             $self->c->model('Edit')->accept($edit);
+
+            # If the editor is a beginner who can now lose the flag, do that
```

Comment:
Hmm, maybe this should be moved into `Edit::accept`? For example, if an auto-editor approves a beginner's edit.

Reply:
Oh, that's fair, I didn't consider that but those should also count. Will move it.

Reply:
Hmm, but `accept` is also run for auto-edits, so we shouldn't *always* run this there. I guess we can pass a parameter for whether it should count or not.

Reply:
Can't you just check `$edit->auto_edit`?